### PR TITLE
Release prep

### PR DESCRIPTION
**Pull request by trlinkin**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,23 +1,79 @@
 {
-  "name": "mrpuppet-tagmail",
-  "version": "0.1.3",
-  "author": "mrpuppet",
-  "summary": "A report processor plugin for re-enabling the tagmail feature in PE 3.7 onwards. Use this to have Puppet send emails based on tags",
+  "name": "puppetlabs-tagmail",
+  "version": "1.0.0",
```
Comment (hunner) [inline metadata.json]: This hasn't gone through QA yet. Should we do something other than a 1.0.0 first until it gets reviewed?

Comment (trlinkin) [inline metadata.json]: Possibly. I forgot we had our own internal QA process for this stuff. A  < 1.0.0 release would be fine.  

--  
Tom Linkin
Professional Services Engineer
http://puppetlabs.com/
twitter: @trlinkin

File: metadata.json [hunk 2]
```diff
@@ -1,23 +1,79 @@
...
+  "author": "puppetlabs",
+  "summary": "This module provides a report processor that sends events to email addresses based on tagging of resources",
   "license": "Apache-2.0",
+  "source": "git://github.com/puppetlabs/puppetlabs-tagmail.git",
+  "project_page": "http://links.puppetlabs.com/tagmail-module",
```
Comment (hunner) [inline metadata.json]: This link doesn't point anywhere. Do we want to pre-seed it to http://forge.puppetlabs.com/puppetlabs/tagmail ?

File: metadata.json [hunk 3]
```diff
@@ -1,23 +1,79 @@
...
+  "issues_url": "https://github.com/puppetlabs/puppetlabs-tagmail/issues",
```
Comment (hunner) [inline metadata.json]: This should be https://tickets.puppetlabs.com/browse/MODULES

File: metadata.json (hunk 2 above)
Comment (hunner) [inline metadata.json]: Actually no, we usually use github url so https://github.com/puppetlabs/puppetlabs-tagmail/

File: metadata.json [hunk 4]
```diff
@@ -1,23 +1,79 @@
...
+  {
+    "name": "puppet",
+    "version_requirement": ">= 3.7.0"
+  }
   ],
-  "source": "https://github.com/anokun7/tagmail",
-  "project_page": "https://github.com/anokun7/tagmail",
-  "issues_url": "https://github.com/anokun7/tagmail/issues"
-}
+  "description": "This module provides a report processor that sends events to email addresses based on tagging of resources",
```
Comment (hunner) [inline metadata.json]: This seems out of order. If you want to get the correct ordering, it's easiest to do a `puppet module build` and then `cp pkg/<tab>/metadata.json metadata.json` to let PMT reformat the json.

**Comment by hunner** [comment]
@trlinkin If you're out of the game, I can just patch the above for you. I'll do so by 8am PST tomorrow if I don't hear back :).

File: metadata.json (hunk 2 above)
Comment (trlinkin) [inline metadata.json]: Yeah, I was following the lead from other internal modules, figured we'd resolve later.

File: metadata.json (hunk 3 above)
Comment (trlinkin) [inline metadata.json]: Ah! Right.

File: metadata.json (hunk 4 above)
Comment (trlinkin) [inline metadata.json]: I'll review the ordering. I didn't see anything suggesting there was an order for the metadata.json file of any significance, but I suppose the description belongs up top when I think about it.

**Comment by trlinkin** [comment]
These changes were included in PR #4

**Pull request closed without merging**